### PR TITLE
Relocate arms restoring copy&paste mess to a new method in the new class classes.items.MutationsHelper

### PR DESCRIPTION
**Pull request by Stadler76**
I've found, that the code snippets for restoring arms only handles
feather and chitin arms and ignores other arms. This is being used
multiple times in Mutations.as and can be a pita to fix, when new
armType are being added to the code (e. g. ARM_TYPE_PREDATOR, for which
I plan to file a PR later).
Even ARM_TYPE_SALAMANDER is missing in that part. I have included this
into restoreArms(...), since I see that as a bug/being unintended. I
assume that this goes without asking Savin. To be precise: I doubt that
Savin wants, that several races would keep salamander arms, while other
arms are being restored.
If this is accepted I plan to file more PRs for other non-DRY-coding
parts of Mutations.as (starting with feather hair removal)
I have relocated this into a new class to keep Mutations.as mostly clean
of these helper methods and because IMHO its easier to keep track of
them instead of bloating Mutations.as with more and more of these
functions.

Testing has been done for hummanus, goblin ale and kanga fruit.
The param keepArms plays nicely, too although I'm not using it atm

TL;DR: slowly change Mutations.as towards more following the DRY(Don't
Repeat Yourself)-coding-principle, so future mutations will be easier to
implement.

### CODE REVIEW

File: classes/classes/Items/Mutations.as [hunk 1]
```diff
@@ -4,10 +4,8 @@
 	import classes.GlobalFlags.kFLAGS;
 	import classes.Scenes.Areas.Forest.KitsuneScene;
 
-	public final class Mutations extends BaseContent
+	public final class Mutations extends MutationsHelper
 	{
-		include "../../../includes/appearanceDefs.as";
-
```
Comment (Stadler76) [inline classes/classes/Items/Mutations.as]: Moved that to classes.items.MutationsHelper

File: classes/classes/Items/MutationsHelper.as [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	 */
+	public class MutationsHelper extends BaseContent 
+	{
+		include "../../../includes/appearanceDefs.as";
+
+		public function MutationsHelper() 
+		{
+		}
+
+		public function restoreArms(changes:Number, changeLimit:Number, keepArms:Array = null):Number
```
Comment (Stadler76) [inline classes/classes/Items/MutationsHelper.as]: Like I said: restoreArms for starters and keepArms is for possible future use.
If this is accepted the next PR will likely be for remove feather hair.
